doc/conf.py

```diff
@@ -146,9 +147,10 @@
 smartquotes = False
 
 # Custom sidebar templates, maps document names to template names.
-html_sidebars = {
-    "**": ["localtoc.html", "globaltoc.html", "relations.html", "sourcelink.html"]
-}
+# Use Default Furo Sidebar
+# html_sidebars = {
+#     "**": ["localtoc.html", "globaltoc.html", "relations.html", "sourcelink.html"]
```

Comment:
What is the difference here ? It seems localtoc and globaltoc could be replaced by something generic from furo, but I'm not so sure for relatons and sourcelink.

Reply:
`localtoc.html` will be replaced with content sidebar
![image](https://user-images.githubusercontent.com/55311527/161356585-9e1e6e46-c11a-4b66-9d97-4f0592c85214.png)

`globaltoc.html` will be replaced with generic `furo` left sidebar:
![image](https://user-images.githubusercontent.com/55311527/161356628-e4e4ff33-5556-420f-b0c4-310b6d6ed349.png)

Reply:
`relation.html` will be replaced with generic `furo` navigation next and previous button:
- furo
![image](https://user-images.githubusercontent.com/55311527/161356830-a1bacf3c-9366-4a96-827a-450a12745bd2.png)
- python-docs-theme
![Uploading image.png…]()

Reply:
`sourcelink.html` will be replaced with edit on GitHub button (little pen icon), and because `pylint` uses `readthedocs`. Edit source file can also be seen in RTD edit/download/version options
![image](https://user-images.githubusercontent.com/55311527/161357474-7266b372-675a-499d-817f-3a57c0e30ada.png)

Reply:
[Furo sidebar docs](https://pradyunsg.me/furo/customisation/sidebar/)